### ISSUE TITLE
Linking failed for ANTSIntegrateVelocityField

### ISSUE THREAD

**Issue by gdevenyi** [stnava/ANTs#204]
Building latest commit (as of 2015.07.10) with pull request #198 integrated to enable MINC support.

```
Linking CXX executable /opt/quarantine/ANTs/git/src/build/bin/ANTSIntegrateVelocityField
/opt/quarantine/ANTs/git/src/build/lib/libl_ANTSIntegrateVelocityField.so.2.1.0: undefined reference to `itk::MINCTransformIOFactoryRegister__Private()'
collect2: ld returned 1 exit status
make[5]: *** [/opt/quarantine/ANTs/git/src/build/bin/ANTSIntegrateVelocityField] Error 1
make[4]: *** [Examples/CMakeFiles/ANTSIntegrateVelocityField.dir/all] Error 2
make[4]: *** Waiting for unfinished jobs....
```

I suspect it may be related to recent commits adding xfm/minc support

**Comment by vfonov** [comment]
That's a problem with ITK: http://review.source.kitware.com/#/c/19972/

will be fixed in the next release I suppose. Right now I am applying this patch when building ITK: https://raw.githubusercontent.com/BIC-MNI/minc-toolkit-v2/develop/cmake-modules/0001-BUG-Missing-ITKIOMINC_EXPORT-on-__Private.patch

**Comment by gdevenyi** [comment]
Cool, it says merged so I think I can updated the ITK commitid for ANTS to get it building. Thanks @vfonov

**Comment by gdevenyi** [comment]
Fixed by #205
